12_upgrade/05_caution.txt

```diff
@@ -1,24 +1,10 @@
 === Caution cases
 
-==== Upgrading from Rudder 2.11
+==== Compatibility between Rudder server 3.1 and older agent versions
 
-Rudder 2.11.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* are fully compatible with Rudder server 3.0.*, so it is not necessary to update your agents to 3.0.*.
+Rudder 2.11.* and 3.0.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* and 3.0.* are fully compatible with Rudder server 3.1.*, so it is not necessary to update your agents to 3.1.*.
 
```

Comment:
Can you add a line to explain older version case: 
"To upgrade from oldest version, you have to upgrade to 3.0 or 2.11 before upgrading to 3.1"